### ISSUE TITLE
Wrong version constraints in composer.json

### ISSUE THREAD

**Issue by stof** [l3l0/php-travis-client#1]
Just for your information, your version constraint for the deps is wrong: For Buzz, you force using the latest master, and for Doctrine Common you allow any version higher than 2.2 Both of these constraints are valid only if you know the future as it means that a release of your library done now would be compatible with any new version of Common, including a 4.3 release done in 7 years. And for Buzz, it even means that you forbid existing releases.
Basically, you should always have an upper bound int he constraint. A lower bound is of course not required as knowing the past releases is generally possible.

PS: if you really know the future, please close the issue as invalid :smile:
